Features/Intel/OutOfBandManagement/IpmiFeaturePkg/GenericIpmi/Dxe/IpmiInit.c

```diff
@@ -229,6 +229,7 @@ Routine Description:
 } // GetSelfTest()
 
 EFI_STATUS
+EFIAPI
 GetDeviceId (

```

Comment:
EFIAPI should only be added to external APIs or var-arg functions.
GetDeviceId is an internal function.
You should remove "EFIAPI" from .h file.